src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -7,28 +7,33 @@ import {AccordionContext} from '~/components/Accordion/Accordion.context';
 
 export const AccordionItem = ({id, label, icon, onClick, children, className, ...props}) => {
```

Comment:
Add callbacks for transition handling - onItemOpening / onItemOpened on Accordion before and after transition , also check what to do with onSetOpenedItem  (currently called before transition)